demo/vl-upload.html

```diff
@@ -10,6 +10,10 @@
   </head>
 
   <body>
+  	<button id="clear-all" onclick="document.querySelectorAll('vl-upload').forEach(upload => upload.clear());">
+  		Clear all
+  	</button>
```

Comment:
Aangezien de rest van de pagina in het NL is zou ik dit ook in het NL schrijven en misschien ook wat verduidelijken (wat is "all"?).